### PR TITLE
POL-303 - AWS RDS resource browser changes

### PR DESCRIPTION
**Pull request by gopinathmahakud1**
Resource browser changes

### Description

Added all changes for the resource browser 

### Issues Resolved

Below resources added for the resource browser
- db_instance 
- security_groups

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

Below are the screenshot tested using Postman.

image

image

### CODE REVIEW

File: aws/rs_aws_rds/aws_rds_plugin.rb [hunk 1]
```diff
@@ -618,8 +714,21 @@
     action "list" do
       verb "POST"
       path "/?Action=DescribeDBSecurityGroups"
+	  field "page_size" do
+          type 'string'
```
Comment (cdwilhelm) [inline aws/rs_aws_rds/aws_rds_plugin.rb:718]: indentation

File: aws/rs_aws_rds/aws_rds_plugin.rb [hunk 2]
```diff
@@ -618,8 +714,21 @@
...
+      end
+     pagination $rds_pagination		  
     end
 
+    polling do
+      field_values do
+      page_size $page_size	  
+      end
+      period 60
+	  action 'list'
```
Comment (cdwilhelm) [inline aws/rs_aws_rds/aws_rds_plugin.rb:730]: indentation

File: aws/rs_aws_rds/aws_rds_plugin.rb [hunk 3]
```diff
@@ -584,7 +663,24 @@
...
+    end
+
+    output 'region' do
+    end
+
+    output 'state' do
+    end
+
+    output 'tags' do
+    end
```
Comment (cdwilhelm) [inline aws/rs_aws_rds/aws_rds_plugin.rb:682]: do/end not needed if body_path isn't used.  Also if outputs without body_path can all be on one line.
output region, state, tags, ...

File: aws/rs_aws_rds/changelog.md [hunk 4]
```diff
@@ -1,27 +1,31 @@
-RDS Plugin changelog
+# Changelog
+
+## v2.0
+
+- Added new resource browser changes.
```
Comment (cdwilhelm) [inline aws/rs_aws_rds/changelog.md:5]: Support for Resource Browser

File: aws/rs_aws_rds/aws_rds_plugin.rb [hunk 5]
```diff
@@ -1,20 +1,66 @@
-name 'aws_rds_plugin'
+name 'Amazon RDS'
```
Comment (cdwilhelm) [inline aws/rs_aws_rds/aws_rds_plugin.rb:1]: filename should be changed to aws_rds.plugin

File: aws/rs_aws_rds/aws_rds_plugin.rb [hunk 6]
```diff
@@ -331,6 +377,25 @@
...
+
+    output 'region' do
+	 body_path 'AvailabilityZone'
+    end
+
+    output 'state' do
+      body_path 'DBInstanceStatus'
+    end
+
+    output 'tags' do
```
Comment (cdwilhelm) [inline aws/rs_aws_rds/aws_rds_plugin.rb:396]: tags can be added to the other output list.

File: aws/rs_aws_rds/aws_rds_plugin.rb [hunk 7]
```diff
@@ -540,6 +605,12 @@
     action "list" do
       verb "POST"
       path "/?Action=DescribeDBInstances"
+	  field "page_size" do
+          type 'string'
```
Comment (cdwilhelm) [inline aws/rs_aws_rds/aws_rds_plugin.rb:609]: indentation

**Review by rshade — APPROVED** [review]
LGTM

**Pull request merged**